filesystem_service/requirements.txt

```diff
@@ -4,3 +4,4 @@
 #
 #    pip-compile --no-index --output-file requirements.txt requirements.in
 #
+inotify
```

Comment:
It is supposed to be added to `requirements.in` and using `pip-tools` added to `requirements.txt` with a pinned version.